csv2mtx.py

```diff
@@ -141,7 +141,8 @@ def __init__(self,
     def get_rows(self) -> Iterable[List[str]]:
         first = True
         for path in self.filepaths:
-            cell = pd.read_csv(path, sep=self.delimiter, compression='infer', header=None, comment='#')
+            # String conversion needed since whitespace is handled in superclass
+            cell = pd.read_csv(path, sep=self.delimiter, compression='infer', header=None, comment='#').astype(str)
```

Comment:
It feels like you're lumping in other changes as well. That's ok in a pinch, but should be stated somewhere.